grammars/asciidoc.cson

```diff
@@ -30,7 +30,7 @@
   #
   #   === Level 2 Section
   {
-    'begin': '^={1,6}\\p{Blank}+'
+    'begin': '^(={1,6}|[#]{1,6})([ \\t]+)'
```

Comment:
This can be written more concisely as:

```
'^((?:=|#){1,6})([ \\t]+)'
```

We may even one to do a lookahead for at least one non-space character:

```
'^((?:=|#){1,6})([ \\t]+)(?=\S)'
```